packages/kbn-journeys/journey/journey_ftr_config.ts

```diff
@@ -93,6 +93,8 @@ export function makeFtrConfigProvider(
           `--telemetry.labels=${JSON.stringify(telemetryLabels)}`,
           '--csp.strict=false',
           '--csp.warnLegacyBrowsers=false',
+          `--xpack.fleet.packages.0.name=cloud_security_posture`,
+          `--xpack.fleet.packages.0.version=1.2.8`,
```

Comment:
For installing package's dependencies.

Reply:
Once https://github.com/elastic/kibana/pull/151277 will be merged, we will be able to have our own configuration.